frontend/src/components/user/OpportunityCompetitionStart.tsx

```diff
@@ -13,7 +13,15 @@ export const OpportunityCompetitionStart = ({
   opportunityTitle: string;
 }) => {
   const { checkFeatureFlag } = useFeatureFlags();
-  const openCompetitions = selectOpenCompetitions({ competitions });
+  const openCompetitions = competitions.reduce<Competition[]>(
```

Comment:
[nit] this could be filter
```suggestion
  const openCompetitions = competitions.filter(({ is_open }) => is_open);
```